mutation/mutation_fragment_v2.hh

```diff
@@ -370,6 +370,31 @@ private:
 
 std::ostream& operator<<(std::ostream&, mutation_fragment_v2::kind);
 
+template <> struct fmt::formatter<mutation_fragment_v2::kind> : fmt::formatter<std::string_view> {
+    template <typename FormatContext>
+    auto format(const mutation_fragment_v2::kind& k, FormatContext& ctx) const {
+        string_view name = "UNEXPECTED";
+        switch(k) {
```

Comment:
might want to add a space after `switch`.

Reply:
Done